src/components/SelectDate.vue

```diff
@@ -80,17 +84,37 @@
 </template>
 
 <script>
+import { displayApi } from "../databazeFilmy";
+
 export default {
   name: "SelectDate",
   data() {
     return {
-      chosenDay: "today"
-      /*options:[
-      {text: "dnes", value: "všechna kina dnes"},
-      {text: "zítra", value: "všechna kina zítra"},
-      {text: "dalších 7 dní", value: "všechna kina dalších 7 dní"},
-      {text: "vše", value: "celý program všech kin"},]*/
+      chosenDay: "today",
+
+      apiData: fetch(
+        "https://api.apify.com/v2/datasets/mF3iLaWJ6zq5fUCSy/items?format=json&clean=1"
+      )
+        .then(resp => resp.json())
+        .then(json => {
+          console.log(json);
+          return json;
+        })
+        .then(json => displayApi)
```

Comment:
Tady tu funkci nevolate... je treba pridat zavorky `displayApi()`.

Navic `displayApi` je z souboru `databazeFilmy`.. to nechceme.. chteli bychom zavolat tu funkci `displayApi` z radku 115 tady.

Tu zavolame takhlenc: `json => this.displayApi(json)`